webroot/js/toolbar-app.js

```diff
@@ -130,6 +130,17 @@ Toolbar.prototype = {
 	},
 
 	bindNeatArray: function() {
+        var sort_button = this.content.find('.neat-array-sort');
+        var _this = this;
+        sort_button.click(function() {
+            if ($(this).attr('checked')) {
+                document.cookie = 'debugKit_sort=; expires=Thu, 01 Jan 1970 00:00:01 GMT; path=/';
```

Comment:
http://www.w3.org/TR/DOM-Level-2-HTML/html.html#ID-8747038

"When this attribute is set, the value it is set to should be a string that adheres to the cookie non-terminal of [IETF RFC 2965]; that is, it should be a single name-value pair followed by zero or more cookie attribute values."